File: java-sdk/java-sdk-test/src/test/java/io/kestra/example/CommonTestSetup.java
package io.kestra.example;

import io.kestra.sdk.KestraClient;

public class CommonTestSetup {
    public static String MAIN_TENANT = "main";

    public static KestraClient kestraClient() {
        return KestraClient.builder()
            .basicAuth("root@root.com", "Root!1234")
            .url("http://localhost:9901")
            .build();
    }
}

File: javascript-sdk/test_javascript_sdk/CommonTestSetup.js
import KestraClient from "../src/KestraClient";

export const host = "http://localhost:9903"
export const username = "root@root.com"
export const password = "Root!1234"
export const MAIN_TENANT = "main";

export function kestraClient() {
    return new KestraClient(host, null, username, password);
}
export function randomId() {
    return Math.random().toString(36).substring(2, 10);
}

export function randomEmail() {
    return randomId() + '@example.com';
}
